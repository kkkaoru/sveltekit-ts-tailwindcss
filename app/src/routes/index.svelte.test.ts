import { render } from '@testing-library/svelte';
import Index from './index.svelte';

describe('top page', () => {
  it('should be matched snapshot', () => {
    const { container } = render(Index);
    expect(container).toMatchSnapshot();
  });
});
