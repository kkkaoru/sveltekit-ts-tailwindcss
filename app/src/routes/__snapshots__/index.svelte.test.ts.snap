// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`top page should be matched snapshot 1`] = `
<body>
  <div>
    <h1>
      This is a example about svelte kit
    </h1>
     
    <img
      alt="thumbs up"
      class="w-32"
      decoding="async"
      height="256"
      loading="lazy"
      src="https://twemoji.maxcdn.com/v/latest/svg/1f44d.svg"
      width="256"
    />
     
    <p>
      Visit 
      <a
        href="https://kit.svelte.dev"
      >
        kit.svelte.dev
      </a>
      
  to read the documentation
    </p>
  </div>
</body>
`;
